warehouse/oidc/models/github.py

```diff
@@ -235,6 +241,25 @@ def publisher_url(self, claims=None):
             return f"{base}/commit/{sha}"
         return base
 
+    def publisher_verification_policy(self, claims):
```

Comment:
Nitpick: docstring explaining the exact inner policy being constructed would be good!

Reply:
fixed